.github/workflows/clippy-check.yml

```diff
@@ -1,12 +0,0 @@
-on: push
-name: Clippy check
-jobs:
-  clippy_check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - run: rustup component add clippy
-      - uses: actions-rs/clippy-check@v1
```

Comment:
It seems the `clippy-check` action (to annotate code in github) is not supported anymore :/
Not sure this is a problem, since it is checked in CI - and not sure anyone was really using it.

Reply:
Yeah :-(

I couldn't find a well maintained replacement for this but arrived to the same conclusion: clippy is being run in the main workflow so we're just losing the fancy annotations. That's a shame, but since this action is unmaintained and also doesn't seem to work for forks it felt like the right trade-off to make.

Reply:
Agreed, let's drop this until we have a real use for it and a maintained upstream